package.json

```diff
@@ -56,7 +56,7 @@
     "@sapphire/async-queue": "^1.5.0",
     "@types/node-fetch": "^2.6.2",
     "@types/ws": "^8.5.3",
-    "discord-api-types": "^0.33.5",
+    "discord-api-types": "^0.37.15",
```

Comment:
this is actually a breaking change

Reply:
serius?

Reply:
u can read more https://github.com/discordjs/discord.js/pull/8795#issuecomment-1295492373

Reply:
Then okey